CHANGELOG.md

```diff
@@ -21,6 +21,13 @@ as the `FSUNDenseMatrix_Data`, `FSUNBandMatrix_Data`, `FSUNSparseMatrix_Data`,
 Compiling and running code that uses the SUNDIALS Fortran interfaces with
 bounds checking will now work. 
 
+A new capability to keep track of memory allocations made through the `SUNMemoryHelper`
+classes has been added. Memory allocation stats can be accessed through the
+`SUNMemoryHelper_GetAllocStats` family of functions. See the documentation for
```

Comment:
Now that this has been pared down to one required operation as part of the class, I would no longer call it a "family of functions".
```suggestion
`SUNMemoryHelper_GetAllocStats` function. See the documentation for
```